functests/utils/namespaces/namespaces.go

```diff
@@ -20,3 +25,29 @@ func WaitForDeletion(cs *testclient.ClientSet, nsName string, timeout time.Durat
 		return false, nil
 	})
 }
+
+// Create creates a new namespace with the given name.
+// If the namespace exists, it returns.
+func Create(namespace string, cs *testclient.ClientSet) {
+	_, err := cs.Namespaces().Create(&k8sv1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: namespace,
+		}})
+
+	if k8serrors.IsAlreadyExists(err) {
+		return
+	}
+
+	Expect(err).ToNot(HaveOccurred())
+}
+
+// Clean cleans all dangling objects from the given namespace.
+func Clean(namespace string, cs *testclient.ClientSet) {
+	_, err := cs.Namespaces().Get(namespace, metav1.GetOptions{})
+	if err != nil {
```

Comment:
we need to check here if the error is because the namespace is not exist and not other errors.